src/app/layouts/ModalWindowContent.js

```diff
@@ -0,0 +1,94 @@
+import { Button } from '../components/Button';
+import { render } from '../rendering';
+
+export const ModalWindowContent = (
+  playerAnswers,
+  computerAnswers,
+  onWindowClose,
+) => {
+  const content = document.createElement('div');
+  content.classList.add('content');
+  content.id = 'content';
+
+  const gameOver = render({
+    component: document.createElement('div'),
+    inside: content,
+    withClasses: 'content__gameOver',
+  });
+  gameOver.textContent = 'game over';
+
+  const playerResult = correctAnswersCounter(playerAnswers);
+  const playerAnswersQuantity = playerAnswers.length;
+  const computerResult = correctAnswersCounter(computerAnswers);
+  const computerAnswersQuantity = computerAnswers.length;
+  const finalResults = render({
+    component: document.createElement('div'),
+    inside: content,
+    withClasses: 'content__finalResults',
+  });
+  //we can also use variable for game time length
+  finalResults.textContent = `The force is strong in you young Padawan! During 1 minute you have answered ${playerResult} / ${playerAnswersQuantity} questions and Computer quessed ${computerResult} / ${computerAnswersQuantity}.`;
+
+  function correctAnswersCounter(answers) {
+    if (answers) {
+      const correctAnswers = answers.filter((elem) => elem.isCorrect);
```

Comment:
A jedną linijką da radę :) ? 
```
function correctAnswersCounter(answers) {
    return answers?.filter((elem) => elem.isCorrect)?.length;
  }
  ```

Reply:
o to nawet nie wiem czy bym wpadł, że po znaku zapytania mogę dalej na zmiennej/parametrze coś wywołać. Muszę zapamiętać!

Reply:
Bez znaku zapytania wywali Ci, ze jest undefined. Caly operator to `?.`oznacza :wywloaj, tylko jak nie undefined.

Reply:
No tak tak, to rozumiem. Nie wiedziałem, że po znaku zapytania można sobie coś tam dalej pisać ;-)